### PR TITLE
#GAME-0003 add P1MissileCorvette ammo counter

### PR DESCRIPTION
**Pull request by KajiTetsushi**
Adds the standard ammunition counter to Turanic Raider missile corvettes.

image

### CODE REVIEW

File: src/Game/Select.c [hunk 1]
```diff
@@ -1303,7 +1303,7 @@ void selStatusDraw0(Ship *ship)
         fontMakeCurrent(selGroupFont0);
         fontPrint(x - halfWidth - fontWidth("C") - 1, rect.y1 + 5 - fontHeight(" "), selHotKeyNumberColor, "C");
     }
-    if((ship->shiptype == MissileDestroyer) || (ship->shiptype == MinelayerCorvette))      //ship has missles, so display missile status
+    if((ship->shiptype == MissileDestroyer) || (ship->shiptype == MinelayerCorvette) || (ship->shiptype == P1MissileCorvette))      //ship has missles, so display missile status
```
Comment (KajiTetsushi) [inline src/Game/Select.c:1306]: [M] We need to move away from hardcoding like this and to switch to figuring out which ships have ammunition.

The _Command & Conquer_ titles from _Tiberian Dawn_ to _Red Alert 2: Yuri's Revenge_ are a good example of this: they combine key-value pairs like [`PipScale`](https://www.modenc.renegadeprojects.com/PipScale) and [`Ammo`](https://www.modenc.renegadeprojects.com/Ammo) to display ammunition counter rather than identifying which exact unit has ammo.

tl;dr: declarative > imperative

**Review by KajiTetsushi — COMMENTED** [review]
To create a new epic issue for this concern as it exists in many, many places.
